web/static/js/settings/next-settings.js

```
@@ -44,10 +44,39 @@
 
+  // M55 #2117 (next-only a11y): the pane composes shared section bodies, one of
+  // which (the Users table) wraps its content in a horizontally scrollable
+  // `.overflow-x-auto` div. axe `scrollable-region-focusable` requires a scroll
+  // container to be keyboard-reachable. We can't add the attribute in the shared
+  // template without changing the stable v1 render, so we enhance it here on the
+  // next/ page only: make each scrollable region focusable (tabindex=0) and, when
+  // a section heading is available, expose it as a labelled region (role=region +
+  // localized aria-label from the heading). Idempotent + re-run-safe.
+  function swMakeScrollRegionsFocusable(root) {
+    var regions = list(root, '.overflow-x-auto, .overflow-auto');
+    for (var i = 0; i < regions.length; i++) {
```

Biome (2.5.1)

[error] This var should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)